src/components/NcAppNavigationItem/NcAppNavigationItem.vue

```diff
@@ -859,20 +903,21 @@ export default {
 }
 
 /* Makes the icon of the collapsible element disappear
-*  When hovering on the root element */
+ * When hovering on the root element
+ */
 .app-navigation-entry--collapsible {
-	//shows the triangle button
+	// hides the triangle button
 	.icon-collapse {
-		visibility: hidden;
+		display: none;
 	}
 	&.app-navigation-entry--no-icon,
 	&:hover, &:focus {
 		a .app-navigation-entry-icon {
 			visibility: visible;
 		}
 		.icon-collapse {
-			//shows the triangle button
-			visibility: visible;
+			// shows the triangle button
+			display: initial;
```

Comment:
I think one of this changes now reverted the idea of https://github.com/nextcloud/nextcloud-vue/pull/4103...

Reply:
cc @Jerome-Herbinet

Reply:
Really? Interesting!

Reply:
I see